listenbrainz_spark/hdfs/tests/test_init.py

```diff
@@ -0,0 +1,101 @@
+import os
+import json
+import tarfile
+import tempfile
+import subprocess
+import unittest
+from datetime import datetime
+from unittest.mock import patch, Mock, call
+
+import listenbrainz_spark
+from listenbrainz_spark import config, utils, schema
+from listenbrainz_spark.hdfs import ListenbrainzHDFSUploader
+from listenbrainz_spark.hdfs.upload import ListenbrainzDataUploader
+
+from pyspark.sql.types import StructField, StructType, StringType
+
+test_listen = {
+            'user_name': 'vansika', 'artist_msid': "a36d6fc9-49d0-4789-a7dd-a2b72369ca45", 'release_msid' : "xxxxxx",
```

Comment:
can we make this cleaner? A key: value pair per line would look much cleaner

Reply:
sure